scripts/push-dockerimage.sh

```diff
@@ -2,6 +2,9 @@
 
 set -e
 
+DOCKERFILE="$0"
```

Comment:
Won't `$0` be the name of the script, not the first argument?

Reply:
yes! thanks, will fix that up.

Any suggestions for how to test this code?

Reply:
There's https://github.com/sstephenson/bats for bash unit tests, but I'm not sure how we'd go about mocking the external calls it makes.
Do we have a test area to upload the images to?